web/src/components/schedule/schedule.styles.js

```diff
@@ -0,0 +1,7 @@
+const scheduleStyles = {
+  schedule: {
+    paddingTop: "68px",
```

Comment:
Increment of 8 preferably

Reply:
Yep - the header is sized differently on different screen sizes too. I'll update this

Reply:
The nav bar doesn't quite follow the 8px increment, so I added a shared `index.styles.js` to have the correct offsets for the Nav Bar, which are 56, 64, and 68.5 px at xs, sm, and md